x/concentrated-liquidity/simulation/sim_msgs.go

```diff
@@ -60,6 +60,114 @@ func RandomMsgCreateConcentratedPool(k clkeeper.Keeper, sim *osmosimtypes.SimCtx
 		SwapFee:                   sdk.NewDecWithPrec(1, 2),
 	}, nil
 }
+func RandMsgCreatePosition(k clkeeper.Keeper, sim *osmosimtypes.SimCtx, ctx sdk.Context) (*cltypes.MsgCreatePosition, error) {
+	// get random pool
+	clPool, poolDenoms, err := getRandCLPool(k, sim, ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	// get random user address with the pool denoms
+	sender, tokens, senderExists := sim.SelAddrWithDenoms(ctx, poolDenoms)
+	if !senderExists {
+		return nil, fmt.Errorf("no sender with denoms %s exists", poolDenoms)
+	}
+
+	// ensure that we have atleast 2 tokens
+	if len(tokens) < 2 {
+		return nil, fmt.Errorf("not enough pool denoms tokens, require 2 got: %d", len(tokens))
+	}
+
+	// Randomize tick values from minTick to maxTick
```

Comment:
```suggestion
	// Retrieve minTick and maxTick from precision factor
``` 

nit: this step isn't actually randomizing